src/support/timefuncs.c

```diff
@@ -29,30 +29,29 @@ extern "C" {
 #endif
 
 #if defined(_OS_WINDOWS_)
-static double floattime(void)
+DLLEXPORT int gettimeofday(struct timeval *tv, void *tzp)
 {
-    struct timeb tstruct;
-
-    ftime(&tstruct);
-    return (double)tstruct.time + (double)tstruct.millitm/1.0e3;
+    struct _timeb tb;
+    errno_t code = _ftime_s(&tb);
```

Comment:
Use `_ftime64_s`?  Especially if we define our own `jl_gettimeofday` as suggested above.